lib/Runtime/ByteCode/ByteCodeWriter.cpp

```diff
@@ -2126,7 +2126,7 @@ namespace Js
         return false;
     }
 
-    void ByteCodeWriter::PatchablePropertyWithThisPtr(OpCode op, RegSlot value, RegSlot instance, RegSlot thisInstance, uint cacheId, bool isCtor, bool registerCacheIdForCall)
+    void ByteCodeWriter::PatchableSuperProperty(OpCode op, RegSlot value, RegSlot instance, RegSlot thisInstance, uint cacheId, bool isCtor, bool registerCacheIdForCall)
```

Comment:
Although this so far is used only for super, I can see a point for leaving it in a generic name here. I will revert it to the old name.

---
In reply to: [132067519](https://github.com/Microsoft/ChakraCore/pull/3336#discussion_r132067519) [](ancestors = 132067519)